0_portfolio_input_check_functions.R

```diff
@@ -647,7 +647,7 @@ check_funds_wo_bbg <- function(fund_data, fin_data) {
     bind_rows(fund_isins_missing_bbg) %>%
     distinct()
 
-  readr::write_csv(fund_isins_missing_bbg, "data/fund_isins_without_bbg_data.csv")
+  write_csv_file(fund_isins_missing_bbg, file = "data/fund_isins_without_bbg_data.csv")
```

Comment:
Wherever possible, we are now using the new `write_csv_file()` and the explicit argumet `file`, which works regardless of the version of readr.